index.bs

```diff
@@ -321,12 +320,17 @@ A <dfn>registry entry</dfn> is a record with the following fields:
 <h4 id="request-link">RequestLink(loader, key)</h4>
 
 <pre emu-alg>
-1. Let _entry0_ be EnsureRegistered(_loader_, _key_).
-1. If _entry0_.[[State]] is "ready", return a new promise fulfilled with _entry_.[[Module]].
+1. Let _entry_ be EnsureRegistered(_loader_, _key_).
+1. If _entry_.[[State]] is "ready", return a new promise fulfilled with _entry_.[[Module]].
 1. Return the result of transforming RequestInstantiate(_loader_, _key_) with a fulfillment handler that, when called with argument _entry_, runs the following steps:
   1. Let _depLoads_ be a new empty List.
   1. For each _pair_ in _entry_.[[Dependencies]], do:
     1. Let _p_ be the result of transforming Resolve(_loader_, _pair_.[[key]], _key_) with a fulfillment handler that, when called with value _depKey_, runs the following steps:
+      1. Let _depEntry_ be EnsureRegistered(_loader_, _depKey_).
+      1. If _depEntry_.[[State]] is "ready", then:
+        1. Let _dep_ be _depEntry_.[[Module]].
+        1. Set _pair_.[[value]] to _dep_.
+        1. Return _dep_.
```

Comment:
I prefer to have this one in, since this is a bug in the current algo, and then do the refactor once #25 is cleared.